nekoyume/Assets/_Scripts/State/RxProps.Arena.cs

```diff
@@ -489,6 +495,14 @@ await _agent.GetStateAsync(purchasedCountDuringIntervalAddress)
                         is Integer iValue
                         ? iValue
                         : 0;
+
+                var arenaAvatarStateAddress =
+                    ArenaAvatarState.DeriveAddress(States.Instance.CurrentAvatarState.address);
+                var arenaAvatarState =
+                    await _agent.GetStateAsync(arenaAvatarStateAddress) is List iValue2
```

Comment:
Is it possible to get state in bulk? With `purchasedCountDuringIntervalAddress` above.